src/App.tsx

```diff
@@ -10,9 +12,17 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route
+              path="*"
+              element={<h1 className="title">Page not found</h1>}
+            />
+
+            <Route path="/people" element={<PeoplePage />}>
+              <Route path=":personHref" element={<PeoplePage />} />
+            </Route>
```

Comment:
The `PeoplePage` component is used for both the parent route `/people` and the child route `:personHref`. This might lead to unexpected behavior if `PeoplePage` is not designed to handle both cases. Consider using a different component for the child route or refactoring `PeoplePage` to handle both scenarios appropriately.